test/test_ops.py

```diff
@@ -79,7 +79,7 @@ def test_forward(self, device, contiguous, x_dtype=None, rois_dtype=None, **kwar
         rois_dtype = self.dtype if rois_dtype is None else rois_dtype
         pool_size = 5
         # n_channels % (pool_size ** 2) == 0 required for PS opeartions.
-        n_channels = 2 * (pool_size ** 2)
+        n_channels = 2 * (pool_size**2)
```

Comment:
I think this is unrelated or due to formatter. But I think the original one is right.

Reply:
Tests were showing ufmt errors for `test/test_ops.py` file, so I used ufmt format for this file. 
Will revert these changes

Reply:
I'm not sure, of formatting. Since, I use pre-commit and it automatically (auto magically?) corrects stuff and then safely commits. You might as well try it :)